cli/src/main/java/com/liferay/blade/cli/command/InitArgs.java

```diff
@@ -91,6 +99,12 @@ public void setUpgrade(boolean upgrade) {
 	@Parameter(description = "[name]")
 	private String _name;
 
+	@Parameter(
+		description = "Specify the profile to use when creating the new Liferay workspace. (default gradle)",
+		names = {"-p", "--profile-name"}
+	)
+	private String _profileName;
+
```

Comment:
Agreed, i think we could remove it.

Reply:
Or we could make it hidden

Reply:
I think it's better to only have one, since they essentially are the same thing anyway - as we are doing this in InitCommand anyway:
```
if (profileName == null) {
    profileName = build;
}
```